internal/internal_workflow_client.go

```diff
@@ -74,7 +73,9 @@ type (
 		identity           string
 		dataConverter      converter.DataConverter
 		contextPropagators []ContextPropagator
-		tracer             opentracing.Tracer
+		workerInterceptors []WorkerInterceptor
+
+		interceptor ClientOutboundInterceptor
```

Comment:
nit: naming: Call this clientInterceptor?

Reply:
I think this stutters, for the same reason we don't call the fields `clientIdentity` or `clientNamespace`.